app/models/rules_factory.rb

```diff
@@ -4,10 +4,18 @@ def initialize(auction)
   end
 
   def create
+    eligibility = if auction.small_business?
```

Comment:
I agree. I think we would basically have a separate column in the database so we could indicate the auction's rules and the population available to participate in it (or the second one might be derived from looking at extant fields already in the DB). But it would be a separate factory.

Instead of calling it Eligibility, maybe we should consider some other term of art like `Pool` or `Population` or such?